content/docs/command-reference/exp/show.md

```diff
@@ -74,7 +74,11 @@ will be generated using the same data from the table.
 - `-A`, `--all-commits` - include experiments derived from all Git commits, as
   well as from the last one. This prints all experiments in the project.
 
-- `-n <num>`, `--num <num>` - show the last `num` commits from HEAD.
+- `--rev <commit>` - "Show experiments derived from the specified revision.
+  Defaults to HEAD if none of `--rev`,`-a`,`-A`,`-T` is specified."
+
+- `-n <num>`, `--num <num>` - show the last `num` commits from the rev
+  specified.
```

Comment:
commits -> experiments? (similar question in https://github.com/iterative/dvc.org/pull/3191#discussion_r787060961)

Reply:
Updated, and paste it here.

Because here what this -n flag does is actually extend the baseline rev to its history commits. And if the -n is -1 then it represents the whole history of a given revision (a tag, a branch, or a commit). And the experiments based on any of the commits from the history will be listed here.

Reply:
p.s. this was also explained under https://github.com/iterative/dvc/pull/7245#discussion_r788133855 🙂